src/wp-includes/class-wp-duotone.php

```diff
@@ -1026,6 +1043,8 @@ public static function set_global_style_block_names() {
 		$block_nodes = $tree->get_styles_block_nodes();
```

Comment:
@swissspidy @joemcgill @felixarntz We should really avoid calling 
```
WP_Theme_JSON_Resolver::get_merged_data()
```
I think it should be safe to replace with 
```
WP_Theme_JSON_Resolver::get_theme_data( array(), array( 'with_supports' => false ) );
```

Reply:
@tellthemachines Do we need all the block data here or just data from theme.json.

Reply:
We need the merged `['default', 'blocks', 'theme', 'custom']` origins because this handles all duotone filters.